packages/cozy-client/src/helpers.spec.js

```diff
@@ -109,3 +109,33 @@ describe('dehydrate', () => {
     )
   })
 })
+
+describe('generateWebLink', () => {
+  it('should generate the right link to a flat cozy', () => {
+    const sharecode = 'sharingIsCaring'
+    const username = 'alice'
+
+    expect(
+      generateWebLink({
+        cozyUrl: 'http://alice.cozy.tools',
+        searchParams: [['sharecode', sharecode], ['username', username]],
+        nativePath: '/n/4',
```

Comment:
(this was for the console.warn)
It's a bit brutal since we already have method to generate an url for an App. 


I think the Note Model should export the `#/n/` path.